Utilities/bootstrap

```diff
@@ -738,9 +771,12 @@ def get_swiftc_path():
         elif platform.system() == 'Darwin':
             return subprocess.check_output(["xcrun", "--find", "swiftc"],
                 stderr=subprocess.PIPE, universal_newlines=True).strip()
-        else:
+        elif platform.system() == 'Linux':
             return subprocess.check_output(["which", "swiftc"],
-            universal_newlines=True).strip()
+                                           universal_newlines=True).strip()
+        elif platform.system() == 'Windows':
+            return subprocess.check_output(["where", "swiftc"],
+                                           universal_newlines=True).strip()
```

Comment:
We could eliminate these two hunks if we factor out another variable for getting the "which" args for a platform.

Reply:
Sure, that seems reasonable.

Reply:
A function is better since we can thread the OS through, if we ever want to improve this script to support cross-compiling scenarios.